services/web/server/src/simcore_service_webserver/resource_usage/_constants.py

```diff
@@ -5,3 +5,5 @@
 MSG_RESOURCE_USAGE_TRACKER_SERVICE_UNAVAILABLE: Final[
     str
 ] = "Currently resource usage tracker service is unavailable, please try again later"
+
+MSG_RESOURCE_USAGE_TRACKER_NOT_FOUND: Final[str] = "Not Found"
```

Comment:
I'd say the same thing here

Reply:
I "cannot find" a better way to answer as @GitHK already did ;-)